src/pages/Main/Main.js

```diff
@@ -7,32 +8,41 @@ import { CATEGORY } from './CategoryListData';
 const Main = () => {
   const [sliders, setSliders] = useState([]);
   const [selectedCategory, setSelectedCategory] = useState(1);
+  const [serviceId, setServiceId] = useState();
+  const history = useHistory();
 
   useEffect(() => {
-    fetch(`http://10.58.0.252:8000/${selectedCategory}`)
+    fetch(`http://10.58.2.233:8000/${selectedCategory}`)
       .then(res => res.json())
       .then(data => setSliders(data.services));
   }, [selectedCategory]);
 
+  const GoToServey = serviceId => {
```

Comment:
```suggestion
  const goToServey = serviceId => {
```
대문자로 시작하는건 컴포넌트 뿐입니다

Reply:
수정 완료!! 😎